test/test_rack_env.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'helpers/svr_in_proc'
+
+# Since there isn't a way to pull the env passed to the app,
+# one of the apps defined in svr_in_proc.rb (rack_env_to_body)
+# returns the env as a response body
+
+class TestRackEnv < ::TestPuma::SvrInProc
+  parallelize_me!
+
+  def test_rack_url_scheme_http
+    # default bind is http
+    rack_url_scheme 'http'
+  end
+
+  def test_rack_url_scheme_https
+    skip_unless :ssl
+    bind_type :ssl
+    rack_url_scheme 'https'
+  end
+
+  def test_rack_url_scheme_user
+    bind_type config: "rack_url_scheme 'user'"
+    rack_url_scheme 'user'
+  end
+
+  def test_rack_url_scheme_aunix_http
```

Comment:
What's the reasoning behind testing this with abstract unix sockets? I don't see why this test is necessary given the other three.

Reply:
I can remove, I added it to show how simple it is to change from one socket type to another.  It is also skipped because when I did the PR, abstract UNIXSockets weren't supported.  I'll remove.